decidim-meetings/app/views/decidim/meetings/meetings/_meetings.html.erb

```diff
@@ -1,5 +1,17 @@
+  <% if @meetings_warning %>
```

Comment:
All the states in this view should be handled directly. Looks like the controller knows too much about view logic. See https://github.com/AjuntamentdeBarcelona/decidim/pull/995/files#r101773425